chromium_src/ui/accessibility/accessibility_features.cc

```diff
@@ -0,0 +1,16 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "src/ui/accessibility/accessibility_features.cc"
+
+#include "base/feature_override.h"
+
+namespace features {
+
+OVERRIDE_FEATURE_DEFAULT_STATES({{
```

Comment:
I'm confused, this feature already appears to be disabled by default, but why would we disable it if it wasn't? We need information about why these changes are made here and in cases like this we definitely need a follow-up issue to address the underlying problem

Reply:
cc: @darkdh

Reply:
nvm on the state, something must have gone wrong the last time I ran sync. I see it now, but the second part still applies

Reply:
The reason of that was already encapsulated in the commit message and I just opened a follow-up [issue](https://github.com/brave/brave-browser/issues/39211)